examples/lusid/transactions/upload-transactions-from-excel.sql

```diff
@@ -0,0 +1,41 @@
+-- ============================================================
+
+-- Description:
+-- 1. In this query, we run an ETL process on some transactions.
+-- 2. First, we load a Excel file of transactions from Drive.
+-- 3. Next, we transform the shape of the transaction data.
+-- 4. Finally we upload the transaction data into LUSID.
+
+-- ============================================================
+
+-- Extract transaction data from LUSID Drive
+@txn_data = 
+use Drive.Excel
+--file=/luminesce-examples/transactions.xlsx
+--worksheet=Sheet 1
```

Comment:
Does the excel workbook have more than 1 sheet? If not this is redundant.

Reply:
It does only have one sheet. Code amended.